.github/workflows/coverity.yml

```
@@ -92,8 +92,17 @@ jobs:
           name: ${{ needs.openvino_download.outputs.ov_artifact_name }}
           path: ${{ env.OV_INSTALL_DIR }}
           merge-multiple: true
+          
+      - name: Cache Coverity Tool
+        id: cache-coverity
+        uses: actions/cache@5a3ec84eff668545956fd18022155c47e93e2684 # v4.2.3
+        with:
+          path: coverity_tool
+          key: coverity-${{ runner.os }}-${{ github.sha }}
+          restore-keys: coverity-${{ runner.os }}
```

I think it should be both before and after the `Download coverity tool` step.
```suggestion
      - name: Cache Coverity Tool
        id: cache-coverity
        uses: actions/cache@5a3ec84eff668545956fd18022155c47e93e2684 # v4.2.3
        with:
          path: coverity_tool.tgz
          key: coverity-${{ runner.os }}-${{ github.sha }}
          restore-keys: coverity-${{ runner.os }}
```